doc/Command-Reference.md

```diff
@@ -4088,6 +4092,170 @@ This command deletes the SNMP Trap server IP address to which SNMP agent is expe
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#management-vrf)
 
+## Muxcable
+
+### Muxcable Show commands
+
+**show muxcable status**
+
+This command displays all the status of either all the ports which are connected to muxcable or any individual port selected by the user. The resultant table or json output will show the current status of muxcable on the port (auto/active) and also the health of the muxcable.
+
+- Usage:
+  ```
+  show muxcable status [OPTIONS] [PORT]
+  ```
+While displaying the muxcable status, users can configure the following fields
+PORT     optional - Port name should be a valid port
+--json   optional - -- option to display the result in json format. By default output will be in tabular format.
```

Comment:
Check the rendered Markdown. There are no line breaks before these lines. Maybe we should make them bullet points?

Reply:
converted them bullet points now, rendered MD looks much better